pkg/targets/reconciler/awscomprehendtarget/adapter.go

```diff
@@ -24,7 +24,6 @@ import (
 	servingv1 "knative.dev/serving/pkg/apis/serving/v1"
 
 	"github.com/triggermesh/triggermesh/pkg/apis/targets/v1alpha1"
-	awscomprehendv1alpha1 "github.com/triggermesh/triggermesh/pkg/apis/targets/v1alpha1"
```

Comment:
Im glad to see these go. I never understood why we did this.

Reply:
It's a leftover from the times when each target had its own controller, `pkg`, etc. I guess.